examples/simple-router-wasm/src/main.rs

```diff
@@ -0,0 +1,51 @@
+//! Run with
+//!
+//! ```not_rust
+//! cd examples && cargo run -p example-simple-router-wasm
+//! ```
+//!
+//! This example shows what using axum in a wasm context might look like. This example should
+//! always compile with `--target wasm32-unknown-unknown`.
+//!
+//! [`mio`](https://docs.rs/mio/latest/mio/index.html), tokio's IO layer, does not support the
+//! `wasm32-unknown-unknown` target which is why this crate requires `default-features = false`
+//! for axum.
+//!
+//! Most "serverless" runtimes use Chrome's V8 engine for executing Javascript, and in our
```

Comment:
There are also other Wasm runtimes using [Wasmtime](https://bytecodealliance.org/articles/wasmtime-1-0-fast-safe-and-production-ready). I would prefer adding more examples or leaving the example out entirely.

Reply:
How about we just strike the first sentence and write:

>Most serverless runtimes expect an exported function that takes in a single request and returns a single response, much like axum's `Handler`.

Reply:
I agree, I don't think we should callout any specific runtimes.